### PR TITLE
Add secondary sources

### PR DESCRIPTION
**Pull request by cfir**
## what
I am adding dynamic object to support secindary sources 
## why
You are not supporting it right now and we need it.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -280,6 +280,23 @@ resource "aws_codebuild_project" "default" {
     }
   }
 
+  dynamic "secondary_sources"{
+      iterator = secondary_source
```
Comment (SweetOps) [inline main.tf:284]: remove pls `iterator` for consistency with other modules

File: main.tf [hunk 2]
```diff
@@ -280,6 +280,23 @@ resource "aws_codebuild_project" "default" {
...
+      for_each = length(var.secondary_sources) > 0 ? var.secondary_sources : []
```
Comment (SweetOps) [inline main.tf:285]: this condition isn't required

File: main.tf [hunk 3]
```diff
@@ -280,6 +280,23 @@ resource "aws_codebuild_project" "default" {
...
+      content {
+        git_clone_depth = secondary_source.value.git_clone_depth
```
Comment (SweetOps) [inline main.tf:287]: need to add fallback to `null` for non-required vars (see https://github.com/cloudposse/terraform-aws-codebuild/blob/974c9d579ca18e4a19a11c9ae1dd858ee76de7b9/main.tf#L324 as example)

**Review by SweetOps — CHANGES_REQUESTED** [review]

**Comment by bcoddens** [comment]
Hey all, what needs to be done to get this integraded ? We need the same feature so if I can help ...

**Comment by jamengual** [comment]
This feature has been added to the module in #87

**Pull request closed without merging**